src/app/cart/cart.component.html

```diff
@@ -0,0 +1,20 @@
+<div class="cart--item-list-container">
+  <ng-container *ngIf="this.getItems().length !== 0">
+    <ul class="item-list cart--item-list">
+      <li *ngFor="let item of this.getItems()">
+        <img
+        class="cart--item-icon"
+        src="{{'../../assets/icons/' + item.id + '.svg'}}"
```

Comment:
We shouldn't need to do relative paths like this. We could just do `"assets/icons" + item.id + ".svg"`